[CTOR-1724]-Refresh applications-protocol-imap doc page (#4499)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-protocol-imap.md

| Service Alias          | Service Template                                | Service Description                             |
|:-----------------------|:------------------------------------------------|:------------------------------------------------|
| Search-Message-Generic | App-Protocol-IMAP-Search-Message-Generic-custom | Check messages in a mailbox with an IMAP filter |

> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.

</TabItem>
</Tabs>

### Collected metrics & status

Here is the list of services for this connector, detailing all metrics and statuses linked to each service.

<Tabs groupId="sync">
<TabItem value="IMAP-Login" label="IMAP-Login">

| Name                    | Unit  |
|:------------------------|:------|
| connection.time.seconds | s     |

</TabItem>
<TabItem value="Search-Message-Generic" label="Search-Message-Generic">

| Name                 | Unit  |
|:---------------------|:------|
| messages.found.count | count |

</TabItem>
</Tabs>

## Prerequisites

> The remote server must have an IMAP service running and available.

Reviewer: ```suggestion
You must have an IMAP service running and available.
```

| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). |                   |             |

</TabItem>
</Tabs>

3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.

## How to check in the CLI that the configuration is OK and what are the main options for?

Once the plugin is installed, log into your Centreon poller's CLI using the
**centreon-engine** user account (`su - centreon-engine`). Test that the connector
is able to monitor a resource using a command like this one (replace the sample values by yours):

```bash
/usr/lib/centreon/plugins/centreon_protocol_imap.pl \
	--plugin=apps::protocols::imap::plugin \
	--mode=search-message \
	--hostname=10.0.0.1 \
	--username='IMAPUSERNAME' \
	--password='IMAPPASSWORD'  \
	--warning=''  \
	--critical='' \

> 	--search=''

Reviewer: FR had --search='xxxxxx'

> 	--search=''

Reviewer 2: ```suggestion
	--username='XXXX' \
	--password='XXXX'  \
	--warning=''  \
	--critical='' \
	--search='xxxxxx'
```
